packages/wonder-blocks-timing/src/hooks/__tests__/use-timeout.test.js

```diff
@@ -42,7 +42,9 @@ describe("useTimeout", () => {
             renderHook(() => useTimeout(action, 1000));
 
             // Act
-            jest.advanceTimersByTime(1000);
+            act(() => {
+                jest.advanceTimersByTime(1000);
+            });
```

Comment:
I didn't realize that `renderHook` was actually rendering a real component.  I had a quick look at its code and it is.  I wonder why we didn't need the `act()` wrappers before. 🤔